file-formats/csnm/README.md

```diff
@@ -0,0 +1,5 @@
+# CSNM File Format
+
+File | Cardinality | Definition | Schema | Example
```

Comment:
Please add all files that are specified here. The provided example includes also two `.csn` files that are actually missing.

Reply:
ok, thanks! actually there could be 0...n files (although 0 files would mean that CSN Model cannot be activated). Therefore I define the cardinality and let only one file as an example